src/custom/pages/Faq/index.tsx

```diff
@@ -322,7 +322,14 @@ export default function Faq() {
               audited
             </ExternalLink>
             . Whilst CowSwap has taken a major step forward in terms of security and stability, as with other crypto
-            protocols or dapps, it&#39;s recommended to use it at <strong>your own risk</strong>.
+            protocols or dapps, your use is at your own risk. <strong> Please review our terms and conditions </strong>
+            <ExternalLink
+              href="https://cowswap.exchange/#/terms-and-conditions"
```

Comment:
What about making this not open in a new page?

Reply:
Sounds good @alfetopito, used the external link right above as a reference for the code.